WordPressKit/ReaderTopicServiceRemote.m

```diff
@@ -237,65 +213,18 @@ - (void)fetchSiteInfoForSiteWithID:(NSNumber *)siteID
 - (RemoteReaderSiteInfo *)siteInfoFromFollowedSiteDictionary:(NSDictionary *)dict
 {
     NSDictionary *meta = [dict dictionaryForKeyPath:@"meta.data.site"];
+    RemoteReaderSiteInfo *obj;
```

Comment:
`obj` is generic. Perhaps, `siteInfo` instead?

Reply:
Done in https://github.com/wordpress-mobile/WordPressKit-iOS/pull/293/commits/d73c2994e45c34a0f1a5b3fdd0efabd969a6b8e2